sound/soc/sof/pm.c

```diff
@@ -319,9 +319,10 @@ int snd_sof_prepare(struct device *dev)
 {
 	struct snd_sof_dev *sdev = dev_get_drvdata(dev);
 	const struct sof_dev_desc *desc = sdev->pdata->desc;
+	int system_target_state = SOF_SUSPEND_S3;
 
 	/* will suspend to S3 by default */
-	sdev->system_suspend_target = SOF_SUSPEND_S3;
+	sdev->system_suspend_target = system_target_state;
```

Comment:
This is for working around unused variable in case !defined(CONFIG_ACPI), right?

But you could...